pyproject.toml

```diff
@@ -43,3 +43,8 @@ mark-parentheses = false
 
 [tool.ruff.lint.isort]
 known-first-party = ["cockpit"]
+
+[tool.vulture]
+ignore_names = [
+   "test[A-Z0-9]*",
```

Comment:
I'd prefer to see `test[A-Z]*` and `test_*` handled separately to avoid pulling in possibly unrelated things.

Reply:
Then we should do that in cockpit as well, I just copied it from there.